charts/consul/templates/api-gateway-gatewayclassconfig.yaml

```diff
@@ -11,6 +11,13 @@ metadata:
     component: api-gateway
 spec:
   consul:
+    {{- if .Values.client.enabled }}
```

Comment:
Think it's worth adding in the comments from the deployment file here too? Either way is fine with me, but maybe just for consistency?

Reply:
Sure! More clarity is better.